### PR TITLE
Fixes #9268: Add takeaways to homepage filters

### LINKED ISSUE #9268: Add filters for new Takeaways
[https://mozilla-hub.atlassian.net/browse/EXP-3113](https://mozilla-hub.atlassian.net/browse/EXP-3113|smart-link) , [https://mozilla-hub.atlassian.net/browse/EXP-3737](https://mozilla-hub.atlassian.net/browse/EXP-3737|smart-link) , and [https://mozilla-hub.atlassian.net/browse/EXP-3738](https://mozilla-hub.atlassian.net/browse/EXP-3738|smart-link) added “QBR notable learning”, “Statistically Significant DaU Gain”, and “Gain Amount” to the Takeaways section.  

We want to add the first 2 (“QBR notable learning” and “Statistically Significant DaU Gain”) as filters to the main screen (at the bottom - as these will be less used - but critical to be able to use).  So it would be a show all “QBR notable learning” and/or “Statistically Significant DaU Gain” filters.  Similar to what was in the older experimenter for other items.

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/EXP-3710)
┆Attachments: <a href="https://mozilla-hub.atlassian.net/rest/api/2/attachment/content/40063">image-20230203-203812.png</a> | <a href="https://mozilla-hub.atlassian.net/rest/api/2/attachment/content/40064">image-20230203-222813.png</a>

### PR DESCRIPTION
**Pull request by eliserichards**
Because

- #9230 and #9188 added two T/F checkboxes for QBR Learnings and Metric Gains to the takeaways section

This commit

- Creates a filter on the homepage for these boolean values so that we can filter the experiments based on the takeaways

----

image

https://github.com/mozilla/experimenter/assets/43795363/bf1cce7e-c499-4388-acc8-e4d59741e6fa

### CODE REVIEW

File: experimenter/experimenter/nimbus-ui/src/types/getConfig.ts [hunk 1]
```diff
@@ -137,6 +137,8 @@ export interface getConfig_nimbusConfig {
   projects: (getConfig_nimbusConfig_projects | null)[] | null;
   types: (getConfig_nimbusConfig_types | null)[] | null;
   statusUpdateExemptFields: (getConfig_nimbusConfig_statusUpdateExemptFields | null)[] | null;
+  takeawaysMetricGain: (boolean | null)[] | null;
+  takeawaysQbrLearning: (boolean | null)[] | null;
```
Comment (eliserichards) [inline experimenter/experimenter/nimbus-ui/src/types/getConfig.ts:141]: These should be 
```
takeawaysMetricGain: (getConfig_nimbusConfig_takeawaysMetricGain | null)[] | null;
takeawaysQbrLearning: (getConfig_nimbusConfig_takeawaysQbrLearning | null)[] | null;
```

File: experimenter/experimenter/nimbus-ui/src/components/PageHome/filterExperiments.ts [hunk 2]
```diff
@@ -82,6 +83,12 @@ const experimentFilters: { [key in FilterValueKeys]: ExperimentFilter<key> } = {
     }
     return targetingConfigMatch;
   },
+  takeaways: (option, experiment) => {
+    return (
+      (experiment.takeawaysQbrLearning && option.value === "QBR_LEARNING") ||
+      (experiment.takeawaysMetricGain && option.value === "METRIC_GAIN")
+    );
+  },
```
Comment (yashikakhurana) [inline experimenter/experimenter/nimbus-ui/src/components/PageHome/filterExperiments.ts:91]: 🎉

**Review by yashikakhurana — APPROVED** [review]
looks 💯 tested locally 👏 👏

**Comment by jaredlockhart** [comment]
Whoaaaa neat!

File: experimenter/experimenter/nimbus-ui/src/components/PageHome/filterExperiments.ts (hunk 2 above)
Comment (jaredlockhart) [reply experimenter/experimenter/nimbus-ui/src/components/PageHome/filterExperiments.ts:91]: Oh we should make these into proper constants at some point.  There's probably a way to pass the enum into gql and have it dump out a ts enum.

**Pull request merged**